app/src/main/java/com/eveningoutpost/dexdrip/utils/Preferences.java

```diff
@@ -1452,6 +1453,10 @@ public boolean onPreferenceChange(Preference preference, Object newValue) {
 
             }
 
+            if (DexCollectionType.getDexCollectionType() != DexCollectionType.LibreReceiver) {
```

Comment:
I would advise wrapping this in a try/catch block in case `libre2settings` is null or other problem.

Reply:
Will have a look on that this evening, and try as you recommend.